src/Polyfills/bootstrap.php

```diff
@@ -58,3 +58,18 @@ function mb_substr( $string, $start, $length = null, $encoding = null ) {
 		return MBString::mb_substr( $string, $start, $length, $encoding );
 	}
 }
+
+if ( ! function_exists( 'mb_internal_encoding' ) ) {
+	/**
+	 * Get internal encoding.
+	 *
+	 * @param string $encoding
+	 *
+	 * @return string|bool
+	 *
+	 * @since x.x.x
+	 */
+	function mb_internal_encoding( $encoding = null ) {
+		return Mbstring::mb_internal_encoding( $encoding );
```

Comment:
PHP isn't case sensitive but it would be nice to use the same case for the class name `MBString`.